fivetran/framework/core/schema/proxy.go

```diff
@@ -0,0 +1,86 @@
+package schema
+
+import (
+    "github.com/fivetran/terraform-provider-fivetran/fivetran/framework/core"
+    resourceSchema "github.com/hashicorp/terraform-plugin-framework/resource/schema"
+    datasourceSchema "github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+)
+
+func ProxySchema() core.Schema {
+    return core.Schema{
+        Fields: map[string]core.SchemaField{
+            "id": {
+                IsId:        true,
+                ValueType:   core.String,
+                Description: "The unique identifier for the proxy within your account.",
```

Comment:
```suggestion
                Description: "The unique identifier for the proxy agent within your account.",
```
Refer to docs: https://fivetran.com/docs/rest-api/proxy-management#response_2